misk-crypto/src/main/kotlin/misk/crypto/InvalidEncryptionContextException.kt

```diff
@@ -0,0 +1,8 @@
+package misk.crypto
+
+import java.security.GeneralSecurityException
+
+class InvalidEncryptionContextException : GeneralSecurityException {
```

Comment:
Do your users want to catch custom exception types? Consider just using something built-in, like `ProtocolException` or just `GeneralSecurityException`.